docs/src/languages/php/extensions.md

```diff
@@ -22,110 +22,110 @@ You can turn on the available ones with the `extensions` key
+| `ftp`             |       |       |       | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `gd`              | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `gearman`         | Avail | Avail | Avail |       |       |       |       |       |       |       |
+| `geoip`           | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |
+| `gettext`         |       |       |       | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `gmp`             | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |
+| `http`            | Avail | Avail |       |       |       |       | Avail | Avail | Avail | Avail |
+| `iconv`           |       |       |       | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `igbinary`        |       |       |       | Avail | Avail | Avail | Avail | Avail | Avail | Avail |
+| `imagick`         | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |       |       |
+| `imap`            | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |
+| `interbase`       | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |       |
+| `intl`            | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `ioncube`         |       |       |       | Avail | Avail | Avail |       |       |       |       |
+| `json`            |       |       | Def   | Def   | Def   | Def   | Def   | Def   | *     | *     |
+| `ldap`            | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |
+| `mailparse`       |       |       |       | Avail | Avail | Avail |       | Avail | Avail | Avail |
+| `mbstring`        |       |       |       | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `mcrypt`          | Avail | Avail | Avail | Avail | Avail |       |       |       |       |       |
+| `memcache`        | Avail | Avail | Avail |       |       |       |       |       |       |       |
+| `memcached`       | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail | Avail |       |
+| `mongo`           | Avail | Avail | Avail |       |       |       |       |       |       |       |
+| `mongodb`         |       |       |       | Avail | Avail | Avail | Avail | Avail |       |       |
+| `msgpack`         |       |       | Avail | Avail | Avail | Avail | Avail | Avail | Avail |       |
+| `mssql`           | Avail | Avail | Avail |       |       |       |       |       |       |       |
+| `mysql`           | Def   | Def   | Def   |       |       |       |       |       |       |       |
+| `mysqli`          | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `mysqlnd`         | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   | Def   |
+| `newrelic`        |       |       | Avail | Avail | Avail | Avail | Avail | Avail | Avail |       |
```

Comment:
Newrelic for php8.1 is not ready yet cf https://github.com/newrelic/newrelic-php-agent/milestone/11

Reply:
I saw your change to the spreadsheet, so that's why the last column is blank (it's just on the next line in narrow screens/this comment here).